src/expression/node/SymbolNode.js

```diff
@@ -68,15 +68,15 @@ export const createSymbolNode = /* #__PURE__ */ factory(name, dependencies, ({ m
       }
     } else if (name in math) {
       return function (scope, args, context) {
-        return name in scope
+        return hasSafeProperty(scope, name)
           ? getSafeProperty(scope, name)
           : getSafeProperty(math, name)
```

Comment:
Hanging stuff of the math object and accessing it with the same machinery as the scope feels… dangerous.

It might be a good idea to have methods for `getScopeProperty` to complement `getSafeProperty`.